src/inspector_io.cc

```diff
@@ -236,6 +236,7 @@ void InspectorIo::WriteCbIO(uv_async_t* async) {
 template<typename Transport>
 void InspectorIo::WorkerRunIO() {
   uv_loop_t loop;
+  loop.data = nullptr;
```

Comment:
But not for long: https://github.com/libuv/libuv/pull/1299

Alternative syntax: `uv_loop_t loop = uv_loop_t()` (or `auto loop = uv_loop_t()` if you don't like to repeat yourself), that zeroes out all fields.